lib/shared/addon/components/cluster-driver/driver-oracleoke/template.hbs

```diff
@@ -208,7 +208,16 @@
               {{input type="text" name="compartment" classNames="form-control" placeholder=(t 'clusterNew.oracleoke.compartmentOCID.placeholder') value=config.compartmentId}}
             {{/input-or-display}}
           </div>
-
+        </div>
+        <div class="row">
+          <div class="col span-4">
+            <label class="acc-label">
+              {{t 'clusterNew.oracleoke.kmsKeyId.label'}}
+            </label>
+            {{#input-or-display editable= isNew value=config.kmsKeyId}}
```

Comment:
I suspect this space in `editable= isNew` will break the page. Can you fix this please?

Reply:
Ahh, good catch.  Thanks.